File: test_src/SuiteScripster.java
/**  ____    __    ____  ____  ____/___     ____  __  __  ____
 *  (  _ \  /__\  (_   )(_  _)( ___) __)   (  _ \(  )(  )(  _ \           Read
 *   )   / /(__)\  / /_  _)(_  )__)\__ \    )___/ )(__)(  ) _ <     README.txt
 *  (_)\_)(__)(__)(____)(____)(____)___/   (__)  (______)(____/   LICENESE.txt
 */
import junit.framework.Test;
import junit.framework.TestSuite;

/**
 * suite to run all base
 * 
 * @author razvanc99
 */
public class SuiteScripster extends TestSuite {
    public static Test suite() {
        TestSuite result = new TestSuite(SuiteScripster.class.getName());

//        result.addTestSuite(com.razie.pub.agent.test.TestMsgService.class);
        
        return result;
    }
}

File: src/test/scala/SuiteScripster.scala
/**
 * Razvan's public code. Copyright 2008 based on Apache license (share alike) see LICENSE.txt for
 * details.
 */

import org.scalatest.junit._
import org.scalatest.SuperSuite

/** main test suite */
class SuitescripsterIWishItWorked extends SuperSuite (
  List (
    new razie.scripster.test.ScalaScriptTest
  )
)

/** TODO this is sooooooooooooo messed up... */
class SuiteScripster () extends junit.framework.TestSuite(classOf[XNadaScripster]) {
  
  // this is where you list the tests...
   addTestSuite(classOf[razie.scripster.test.ScalaScriptTest])
   addTestSuite(classOf[razie.scripster.test.SimpleScriptTest])
   addTestSuite(classOf[razie.scripster.test.ScalaComplexTest])
   
   def test1() = 
     // don't touch this line
     addTest(new junit.framework.TestSuite(classOf[razie.scripster.test.ScalaScriptTest]))
     
}

// this is here to convince eclipse to run as/junit...
class XNadaScripster extends junit.framework.TestCase {
 def testNada : Unit =  {}
}

